### PR TITLE
Cleanup and add some CreditCardAttribute tests

### PR DESCRIPTION
**Pull request by hughbe**

### CODE REVIEW

File: src/System.ComponentModel.Annotations/tests/CreditCardAttributeTests.cs [hunk 1]
```diff
@@ -11,115 +11,94 @@ public class CreditCardAttributeTests
...
+        [InlineData("SomeErrorMessage", "SomeErrorMessageResourceName", null)]
+        [InlineData(null, "SomeErrorMessageResourceName", null)]
+        [InlineData(null, null, typeof(ErrorMessageResources))]
+        public static void Validate_InvalidErrorMessage_ThrowsInvalidOperationException(string message, string resourceName, Type resourceType)
         {
             var attribute = new CreditCardAttribute();
-            attribute.ErrorMessage = null; // note: this overrides the default value
+            attribute.ErrorMessage = message;
+            attribute.ErrorMessageResourceName = resourceName;
+            attribute.ErrorMessageResourceType = resourceType;
```
Comment (stephentoub) [inline src/System.ComponentModel.Annotations/tests/CreditCardAttributeTests.cs]: I'm not sure this is actually testing the same thing as was there previously.  Setting these to null isn't the same as not setting them, e.g. https://github.com/dotnet/corefx/blob/d0dc5fc099946adc1035b34a8b1f6042eddb0c75/src/System.ComponentModel.Annotations/src/System/ComponentModel/DataAnnotations/ValidationAttribute.cs#L148-L157

**Comment by stephentoub** [comment]
Other than my one comment, LGTM.  Thanks for the cleanup.

File: src/System.ComponentModel.Annotations/tests/CreditCardAttributeTests.cs (hunk 1 above)
Comment (hughbe) [inline src/System.ComponentModel.Annotations/tests/CreditCardAttributeTests.cs]: I see - I've fixed this

**Comment by stephentoub** [comment]
Test Innerloop CentOS7.1 Debug Build and Test please
Test Innerloop Ubuntu14.04 Release Build and Test please

**Pull request merged**